src/components/charts/BarChart.tsx

```diff
@@ -196,6 +196,10 @@ const BarChart = () => {
     );
     setIsLoading(false);
   };
+  useEffect(() => {
+    const removeEventListener = submitOnEnter(incrementCounter);
```

Comment:
in `SubmitOnEnter`, you are passing a callback.
But `incrementCounter` is not a callback.
Please explain.

Reply:
im not getting entirely on what u meant over here  . const incrementCounter = () => {
    setSubmitCounter(submitCounter + 1);
  }; im passing this one into submitOnEnter.